src/LDemo/main.cpp

```diff
@@ -15,14 +15,64 @@
 
 #include <LSystem/IO.h>
 
+#include <L3D/Camera.h>
+
 #include <thread>
+#include <string>
 
 using namespace LepusEngine;
 
 #ifndef _DEBUG
 #define NDEBUG
 #endif
 
+float fov = 0.f;
+float angleYaw = 0.f, anglePitch = 0.f;
+const float camSpeed = 0.005f;
+double xposLast = -1.0;
+double yposLast = -1.0;
+float deltaTime = 0.f;
+lepus::gfx::Camera camera;
+
+void scrollCallback(GLFWwindow* window, double xoffset, double yoffset)
+{
+	fov = fmax(1.f, fmin(179.f, fov + (float)yoffset));
+	LepusEngine::ConsoleLogger::Global().LogInfo("", "scrollCallback", "yoffset", (char*)std::to_string(xoffset).append(", ").append(std::to_string(yoffset)).append("FOV: ").append(std::to_string(fov)).c_str());
+}
+
+void mouseCallback(GLFWwindow* window, double xpos, double ypos)
```

Comment:
separate issue: create an App class to provide a neater way for writing demo programs

Reply:
#30